pymc_marketing/mlflow.py

```diff
@@ -528,12 +928,29 @@ def autolog(
     """
     arviz_summary_kwargs = arviz_summary_kwargs or {}
 
+    def patch_compute_log_likelihood(compute_log_likelihood):
+        """Patch the compute_log_likelihood function to log LOOCV metrics.
+
+        We don't need to call this directly again in MMM.fit patch, since that function calls
+        pm.sample() internally anyway.
```

Comment:
sample never calls compute_log_likelihood to my understand. CC @juanitorduz 
Nor would we want it to

Do we need to adjust any logic to fit accordingly?

Reply:
Ok I've updated my comments accordingly. I think with the `if log_loocv and "log_likelihood" not in idata.groups():` line it should be fine, but I'm not totally clear on what that's necessary tbh. Where else is log_likelihood being created? 🤔

Reply:
It is only created when pm.compute_log_likelihood is called and not pm.sample. Ie idata wont have it unless someone calls it